tests/e2e/framework/kubernetes.go

```diff
@@ -90,6 +90,8 @@ const (
 	kubernetesReadinessTimeout        = time.Second * 180
 	kubernetesReadinessInterval       = 200 * time.Millisecond
 	validationWebhookReadinessTimeout = time.Minute
+	istioOperatorTimeout              = time.Second * 300
+	istioOperatorFreq                 = time.Minute
```

Comment:
Seems way to long

Reply:
it wil take 2-3 minutes for the 1st time reconciling. So for sure we leave 5 minutes here.